test/e2e/templates/basic.tpl

```diff
@@ -1,6 +1,9 @@
 {{#> cdnLayout}}
 var options = {
-  'baseUrl': '{{{WIDGET_TEST_SERVER}}}'
+  'baseUrl': '{{{WIDGET_TEST_SERVER}}}',
+  'colors': {
```

Comment:
Since this is an optional config param, I'd suggest **not** including it in the template.

What do you think about using [this type of helper method](https://github.com/okta/okta-signin-widget/blob/master/test/e2e/specs/basic_spec.js#L45-L56) instead?

Reply:
Make sense 👍